contracts/src/interfaces/IChainlinkAggregatorV3.sol

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.0;
```

Comment:
```suggestion
pragma solidity ^0.8.22;
```

Reply:
We've been doing `^0.8.20` since a lot of contracts still use `0.8.20`. Regardless, it's a good catch, and I'll update it.